packages/beacon-node/src/network/peers/discover.ts

```diff
@@ -389,6 +390,10 @@ export class PeerDiscovery {
         addedUnixMs: Date.now(),
       };
 
+      if (this.isPeerCoolingDown(cachedPeer)) {
+        return DiscoveredPeerStatus.peer_cooling_down;
```

Comment:
just found that we also check score state as unhealthy as line 367 above. Maybe replace that line with this check.

Reply:
I moved `isCoolingDown` to check the decay delay period so its possible that the score could be "healthy" but a cool-down was applied because they disconnected us before we downscored them.  I moved the check up with the others and added a comment.